ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type_keys.schema.yml

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+$defs:
```

Comment:
It would be ok to just keep the old schema as it was, and ref to that instead of the def. I think that would be simpler to read and maintain.

Reply:
Can you point me to an example of where that's been done before?  I can then rework this.

Reply:
If you revert the changes to the existing schema, you can just set `$ref: eos_designs#/keys/node_type_keys` under the custom one.

Reply:
Thanks - have fixed in latest commit.